students_proj/src/main/java/com/musala/simple/students/Main.java

```diff
@@ -0,0 +1,171 @@
+package com.musala.simple.students;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+public class Main {
```

Comment:
`Main` class needs formatting corrected, use Eclipse's built-in formatter (`Ctrl + Shift + F`). :) Everything else seems fine! :)